### ISSUE TITLE
Minor issue with `kostkaGelfandTsetlinPatterns'`

### ISSUE THREAD

**Issue by stla** [bkomuves/combinat#1]
Hello,

First, thank you for this excellent package.

I've found a minor issue in the function `kostkaGelfandTsetlinPatterns'` (and consequently `kostkaGelfandTsetlinPatterns`). The problem is that you use the `minimum` function:

```haskell
kostkaGelfandTsetlinPatterns' :: Partition -> [Int] -> [GT]
kostkaGelfandTsetlinPatterns' plam@(Partition lambda0) mu0
  | minimum mu0 < 0                       = []
```
and then an error is thrown when `mu0 = []`.

A possible fix is to use `any (< 0) mu0` instead of `minimum mu0 < 0`.

**Comment by bkomuves** [comment]
Thanks for the report! (for some reason I missed it earlier)

**Comment by bkomuves** [comment]
Fixed by 8c7e4f155106638eabb5323f2b48279498343536
